component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -301,26 +301,17 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
         String contextToken = (String) context.getProperty(EmailOTPAuthenticatorConstants.OTP_TOKEN);
         long generatedTime = (long) context.getProperty(EmailOTPAuthenticatorConstants.OTP_GENERATED_TIME);
         boolean isExpired = isExpired(generatedTime, context);
+
+        AuthenticatedUser authenticatedUser = getAuthenticatedUser(context);
+        if (authenticatedUser == null) {
+            String errorMessage = "Authenticated user is NULL";
```

Comment:
shall we change the message as
"Could not find an Authenticated user in the context"